src/ngx_http_lua_connection.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 2014 Daurnimator
```

Comment:
@daurnimator Will you please state that "I hereby assign copyright in this code to the lua-nginx-module project, to be licensed under the same terms as the rest of the code"? This is just for legal resolution. Thanks for  your understanding.